tutorials/nservicebus-101/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "NServiceBus 101: Messaging Basics"
+suppressRelated: true
+---
+
+In this self-paced course, you will learn the most important fundamentals of how to get working quickly with NServiceBus.
+
+
+## Objectives
+
+In this course we'll build a messaging system using a retail e-commerce domain.
```

Comment:
Like what? This isn't meant to be convince you why to use NServiceBus. You've already decided to give it a try so we're just pointing out that we're talking an e-commerce domain as opposed to building a blog or something else.